src/ui/src/components/blueprints/BlueprintsBlueprint.vue

```diff
@@ -112,12 +112,17 @@
 		<BlueprintToolbar
 			class="blueprintsToolbar"
 			@autogen-click="showAutogen"
+			@deploy="showDeploy"
```

Comment:
You can get rid of the function and simply do
```suggestion
			@deploy="isDeployModalShown= true"
```